.github/actions/run-airbyte-ci/action.yml

```
@@ -79,6 +79,9 @@ inputs:
   python_registry_token:
     description: "Python registry API token to publish python package"
     required: false
+  python_registry_url:
+    description: "Python registry URL to publish python package"
```

```suggestion
    description: "Python registry URL to publish python package"
    default: "https://upload.pypi.org/legacy/"
```
The default value will allow less moving parts and the workflow side.
Why are we using legacy?

---

Fair point, moved it into the default.

It's called a little weird, but by my understanding this is the normal endpoint. It's called "legacy" because of the compatibility: https://warehouse.pypa.io/api-reference/legacy.html#upload-api

> https://warehouse.pypa.io/api-reference/legacy.html#upload-api